### ISSUE TITLE
Errors on MacOS Catalina 10.15.5

### ISSUE THREAD

**Issue by burnsac5040** [roddhjav/pass-import#129]
When installing `pass-import` with `pip` a folder is created in `username/opt/anaconda3/lib/python3.8/site-packages/usr/share/bash-completion/completions/pass-import`.  I'm not sure if there is supposed to be a bash-completion folder in usr/share or usr/local/share.  

When installing with the `git` command, running `make` and then `sudo make install PREFIX=/usr/local` everything works fine until where the last line it shows:

	`error: could not create '/usr/lib/password-store': Operation not permitted
	make: *** [install] Error 1`

If I go into `pass-import/setup.py` and change `lib = Path('/usr', 'lib', 'password-store', 'extensions')` to `lib = Path('/usr/local', 'lib', 'password-store', 'extensions') and then run `sudo make install PREFIX=/usr/local` I get the success message saying "pass-import is installed succesfully" and there is a file `import.bash` in `/usr/local/lib/password-store/extensions`.

However, whenever I run any `pass import ...` commands, I get these errors: 

`Traceback (most recent call last):
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/site-packages/pass_import/__init__.py", line 10, in <module>
    import pass_import.formats  # noqa
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/site-packages/pass_import/formats/__init__.py", line 3, in <module>
    from pass_import.formats.kdbx import KDBX
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/site-packages/pass_import/formats/kdbx.py", line 20, in <module>
    from pass_import.tools import getpassword
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/site-packages/pass_import/tools.py", line 11, in <module>
    import magic
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/site-packages/magic.py", line 61, in <module>
    _open = _libraries['magic'].magic_open
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/ctypes/__init__.py", line 394, in __getattr__
    func = self.__getitem__(name)
  File "/Users/lucasburns/opt/anaconda3/lib/python3.8/ctypes/__init__.py", line 399, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: dlsym(RTLD_DEFAULT, magic_open): symbol not found`

Any help would be appreciated.

**Comment by roddhjav** [comment]
I fixed the base prefix issue in the `setup.py` file. Thanks for the report. Regarding the other issue, it seems to be linked to the magic module that it is not able to import. The issue itself seems to be related to mac. pass-import can use the magic but it is not mandatory, so you can remove it.

**Comment by burnsac5040** [comment]
> I fixed the base prefix issue in the `setup.py` file. Thanks for the report. Regarding the other issue, it seems to be linked to the magic module that it is not able to import. The issue itself seems to be related to mac. pass-import can use the magic but it is not mandatory, so you can remove it.

I tried deleting the git install and trying to install locally and got different errors.  And then I tried to just install it using pip and then imported: 
`from pass_import.managers.dashlane import DashlaneCSV
from pass_import.managers.passwordstore import PasswordStore`

and got this error (relating to magic): `AttributeError: dlsym(RTLD_DEFAULT, magic_open): symbol not found`.

I think that installing it with the git clone puts all the files in the correct place, so I'll do that.
But where do I delete magic from and should I do it before or after running the `make` command?
